assets/src/components/block-mover/draggable.js

```diff
@@ -95,6 +97,15 @@ class Draggable extends Component {
 			return;
 		}
 
+		/*
+		 * On dragging, the browser creates an image of the target, for example, the entire text block.
+		 * But there's already a clone below that's rotated in case the block is rotated,
+		 * and this can create a non-rotated duplicate of that.
+		 * So override this with an empty image.
+		 */
+		const dragImage = new Image();
+		event.dataTransfer.setDragImage( dragImage, 0, 0 );
```

Comment:
This workaround might need more thought, but at least prevents a duplicate, non-rotated image on dragging.

Reply:
Without this workaround (maybe hack):

![duplicate-appears](https://user-images.githubusercontent.com/4063887/58393226-72c0b580-8003-11e9-8ee7-3bdfd6a78c97.gif)